wiki/People/osu!monthly_team/en.md

```diff
@@ -8,7 +8,7 @@ The contributors of [osu!monthly](/wiki/Community/osu!monthly) are active player
 
 ### Main contributors
 
-The people listed below are the core of the osu!monthly team and participate every month in the production of the news posts.
+The people listed below are the core of the osu!monthly team and participate every month in the production of the news post.
```

Comment:
this was intentional (they are indeed multiple posts). it can go either way though, but we're already using "monthly newsletter" (singular, as it should be) at the top of the osu!monthly article.